INSTALL.md

```diff
@@ -63,207 +63,286 @@ cfgov assets
 cfgov start django
 ```
 
+
 ## Stand-alone installation
```

Comment:
Apparently I am. It's weird that the anchor link isn't working for me.

I also wonder, should we reverse the order of stand-alone and vagrant as they appear in the document, since we're not sure if the Vagrant box still works, or even if we want to keep the Vagrant box?